api/coreservice.go

```diff
@@ -1038,11 +1038,10 @@ func (core *coreService) BlockMetas(start uint64, count uint64) ([]*iotextypes.B
 
 	var (
 		tipHeight = core.bc.TipHeight()
-		res       []*iotextypes.BlockMeta
+		res       = make([]*iotextypes.BlockMeta, 0)
 	)
 	if start > tipHeight {
-		log.L().Debug("err in BlockMetas()", zap.Error(status.Error(codes.InvalidArgument, "start height should not exceed tip height")))
-		return nil, nil
+		return nil, status.Error(codes.NotFound, "start height should not exceed tip height")
```

Comment:
should use `codes.InvalidArgument`, same as above 2 places

Reply:
no